tests/old/test_metafeature_extraction.py

```diff
@@ -31,6 +32,6 @@ def test_metafeature_extraction(dataframe):
 
 if __name__ == '__main__':
     
-    for dataset_name in ('38_sick', '196_autoMpg'):
+    for dataset_name in (strings.CLASSIFICATION_DATASET_NAME, '196_autoMpg'):
```

Comment:
Looking in the `byudml.strings` module, I don't see a `CLASSIFICATION_DATASET_NAME` variable.

Reply:
I decided to add this fix to the PR as well to save Erik the hassle.